repl/repl.go

```diff
@@ -0,0 +1,312 @@
+package repl
```

Comment:
Can we change this to `cli` instead? REPL is just one of the functions that tengo CLI does. `cli/cli.go`